packages/ra-ui-materialui/src/input/InputHelperText.tsx

```diff
@@ -18,7 +18,9 @@ const InputHelperText: FunctionComponent<Props> = ({
         <ValidationError error={error} />
     ) : helperText ? (
         <>{translate(helperText, { _: helperText })}</>
-    ) : null;
+    ) : (
+        <></>
```

Comment:
This feels weird. I'd prefer that you return an empty space, as explained in the material-ui documentation:

![image](https://user-images.githubusercontent.com/99944/73255801-209ee800-41c1-11ea-90ee-e78e77306376.png)

https://material-ui.com/api/form-helper-text/#formhelpertext-api

Reply:
ok.
Not fan of the empty space, that feels much weird for me.
But if it is material-ui advice there is no reason to refuse it.

Reply:
As FonctionComponent must return a ReactElement, I need to wrap the empty string in a Fragment